README.md

```diff
@@ -19,7 +19,15 @@ To build and serve the site, you'll need these tools:
 
 ### Setup
 
-Once you've installed the [prerequisites](#prerequisites), get local packages:
+The etcd.io website uses the [Docsy](https://www.docsy.dev) Hugo theme. Install the `git` submodule with the following command:
+
+# pull in the Docsy submodule
+
+```bash
+git submodule update --init --recursive --depth 1
+```
+
+Next, get local packages:
 
 ```bash
 npm install
```

Comment:
We don't have an npm `install` script anymore.

Instead use `get-submodules`.

Reply:
Sry, rushing a bit to head out. Of course we need to keep `npm install`.
Add `npm run get-submodules` after and that'll do the trick.

Reply:
I had forgotten we'd added that npm script. This is correct.